### PR TITLE
Snagging fixes - CSV upload

### PR DESCRIPTION
**Pull request by steventux**
### Context

Design review of CSV upload has suggested some amendments and fixes.

#### Upload form amendments
image

#### Success message
image

#### Amended error for form submission with no file selected
image

#### Error message for non-CSV upload
image

### Link to Trello card

https://trello.com/c/VGKXwQu0/116-snagging-csv-upload

### Checklist

- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by steventux** [comment]
[Snagging - CSV upload](https://trello.com/c/VGKXwQu0/116-snagging-csv-upload)

**Review by AbigailMcP — COMMENTED** [review]
Looking good!

One thing which I realise now wasn't clear on the ticket - rather than the flash message we intended to have a separate 'green panel' success page. Similar to this page the user sees after submitting feedback on refer:

Screenshot 2023-08-10 at 11 57 55

I'm not sure about the content underneath the panel though, one to check with @adamsilver / @johndoates

File: app/views/support_interface/uploads/new.html.erb [hunk 1]
```diff
@@ -1,16 +1,11 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds-from-desktop">
-    <h1 class="govuk-heading-l">Check the Children’s Barred List</h1>
-
-    <h2 class="govuk-heading-m">Upload file</h2>
-
     <%= form_with model: @upload_form, url: [:support_interface, :uploads] do |f| %>
       <%= f.govuk_error_summary %>
```
Comment (AbigailMcP) [inline app/views/support_interface/uploads/new.html.erb:4]:
```suggestion
      <%= f.govuk_error_summary("There’s a problem") %>
```

File: config/master.key.bak [hunk 2]
```diff
@@ -0,0 +1 @@
+626d146b0e45c9c4d740f5f7963eb5ba
```
Comment (AbigailMcP) [inline config/master.key.bak:1]: Do we need this file?

**Review by AbigailMcP — APPROVED** [review]

File: config/master.key.bak (hunk 2 above)
Comment (steventux) [reply config/master.key.bak]: No, added by accident - this isn't a key we ever use in any environment, was generated locally, I've removed now.

**Pull request merged**